app/build.gradle

```
@@ -163,6 +163,32 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
 // Generate Kotlin code for the Focus Glean metrics.
 // -------------------------------------------------------------------------------------------------
 apply plugin: "org.mozilla.telemetry.glean-gradle-plugin"
+apply plugin: "org.mozilla.components.nimbus-gradle-plugin"
+
+nimbus {
+    // The path to the Nimbus feature manifest file
+    manifestFile = "nimbus.fml.yaml"
+    // The fully qualified class name for the generated features.
+    // If the classname begins with a '.' this is taken as a suffix to the app's package name
+    destinationClass = ".nimbus.FocusNimbus"
+    // Map from the variant name to the channel as experimenter and nimbus understand it.
+    // If nimbus's channels were accurately set up well for this project, then this
+    // shouldn't be needed.
+    channels = [
+            focusDebug: "debug",
+            focusNightly: "nightly",
+            focusBeta: "beta",
+            focusRelease: "release",
+            klarDebug: "debug",
+            klarNightly: "nightly",
+            klarBeta: "beta",
+            klarRelease: "release"
+    ]
+    // This is generated by the FML and should be checked into git.
+    // It will be fetched by Experimenter (the Nimbus experiment website)
+    // and used to inform experiment configuration.
+    experimenterManifest = ".experimenter.json"
```

The Experimenter team has requested we use `.yaml`.

```suggestion
    experimenterManifest = ".experimenter.yaml"
```

---

Ah, so this needs to be YAML, as the suggested changes suggests.